assignments/function-conversion.js

```diff
@@ -1,30 +1,29 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => {
+  console.log("Function was invoked!");
+};
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = param => {
+  param;
+};
+anotherFunction("Example");
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
-
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
+let add = (param1, param2) => {
+  return param1 + param2;
+};
+console.log(add(1, 2));
 
+let subtract = (param1, param2) => {
+  return param1 - param2;
+};
```

Comment:
So far so good, however you can take all of your solutions a step further because they only have a single return statement:

```
let subtract = (param1, param2) => param1-param2
```